api/locale/de/LC_MESSAGES/django.po

```diff
@@ -706,155 +738,189 @@ msgid ""
 msgid "Please go to the following page and choose a new password:"
 msgstr "Bitte gehen Sie zu folgender Seite and wählen ein neues Passwort:"
 
 #: api/templates/email_password_reset.html:10
+#, fuzzy
+#| msgid "Your username, in case youve forgotten:"
 msgid "Your username, in case you’ve forgotten:"
 msgstr "Ihr Benutzername:"
 
 #: api/templates/email_quote_done.html:4
-#, python-format
-msgid "Le devis pour la commande n°%(order_id)s a été réalisé."
+#, fuzzy, python-format
+#| msgid "The quote for order n%(order_id)s has been completed."
+msgid "The quote for order n°%(order_id)s has been completed."
 msgstr "Das Angebot für die Bestellung Nr. %(order_id)s wurde erstellt."
 
 #: api/templates/email_quote_done.html:5
 msgid ""
-"Connectez-vous sur le géoshop et confirmez ou annulez votre commande dans la "
-"section \\"
+"Log in to the Geoshop and confirm or cancel your order in the \"My Orders\" "
+"section.\""
 msgstr ""
 "Öffnen Sie den GeoShop und bestätigen oder stornieren Sie Ihre Bestellung im "
 "Abschnitt \\"
 
 #: api/templates/email_user_change.html:3
 msgid ""
 "\n"
-"        <p>Vos demandes de modification de profil ont été envoyées aux "
-"administrateurs du Geoshop.</p>\n"
+"        <p>Your profile update requests have been sent to the Geoshop "
+"administrators.</p>\n"
 "        <p>\n"
-"          Elles seront analysées et reportées dans les jours à venir.\n"
-"          Vous serez averti par email une fois les modifications "
-"effectuées.\n"
+"          They will be reviewed and implemented in the coming days.  \n"
+"          You will be notified by email once the updates are completed.  \n"
 "        </p>\n"
 msgstr ""
 "\n"
-"        <p>Ihre Änderung wurde an die Administratoren des "
-"Geoshops gesendet.</p>\n"
+"        <p>Ihre Änderung wurde an die Administratoren des Geoshops gesendet."
+"</p>\n"
 "        <p>\n"
 "          Diese wird geprüft und baldmöglichst übernommen.\n"
 "          Sie werden per e-Mail informiert, sobald die Änderung durchgeführt "
 "wurde.\n"
 "        </p>\n"
 
-
 #: api/templates/email_user_footer.html:4
-msgid "Cet email est adressé à"
+msgid "This email was sent to"
 msgstr "Diese e-Mail wird gesendet an"
 
 #: api/templates/email_validation_needed.html:6
 msgid ""
-"Votre accord est nécessaire en vue de traiter une commande du géoshop. "
-"Cliquez sur ce lien pour valider ou refuser la commande:"
+"Your approval is required to process an order from the Geoshop. Click on "
+"this link to approve or reject the order:"
+msgstr ""
+"Ihre Zustimmung ist erforderlich zur Bearbeitung einer Bestellung im "
+"GeoShop. Klicken Sie auf die folgende Adresse, um die Bestellung zu "
+"validieren oder abzulehnen:"
+
+#: api/templates/email_validation_needed.html:12
+msgid ""
+"Please note that the client will not receive the data until you take action."
+msgstr ""
+
+#: api/templates/email_validation_needed.html:13
+msgid "If you believe this email was sent to you in error, please contact "
 msgstr ""
```

Comment:
We need a translation.
"Wenn diese E-Mail irrtümlich an Sie gesendet wurde, wenden Sie sich bitte an "